iModelJsNodeAddon/api_package/ts/runTests.mke

```diff
@@ -16,7 +16,7 @@ npmArg =
         %warn *** Skipping tests as we're not running on an Apple Silicon processor ***
         %return
     %else
-        npmCmd = $(SrcRoot)thirdparty/nodejs_16_15_0/bentley/npm.sh
+        npmCmd = npm

```

Comment:
This is harmlessly redundant now.

Reply:
Turns out the npmArg is really only to npm.sh so can be removed entirely. I took a guess, but there may be some work that needs to be done when it's running on the ARM boxes.